cfripper/config/regex.py

```diff
@@ -173,3 +173,18 @@
 - sns:Get*
 """
 REGEX_HAS_STAR_OR_STAR_AFTER_COLON = re.compile(r"^(\w*:)*[*?]+$")
+
+
+"""
+Check that stack name only consists of alphanumerical characters and hyphens.
+Valid:
```

Comment:
Just wanting to confirm a1b2c3 would also be valid? i.e. anything including numbers anywhere?

Reply:
Good point, will add that in the description